parsers/to-react-native/__snapshots__/to-react-native.spec.ts.snap

```diff
@@ -189,6 +189,181 @@ export default theme;
 "
 `;
 
+exports[`To React Native Should support TypeScript 1`] = `
```

Comment:
It seems the snapshot is not up to date on CI

Reply:
@nclsndr Hmm... mind checking again? Should be up-to-date.